pkgs/top-level/all-packages.nix

```diff
@@ -16929,13 +16929,8 @@ in
   };
   coq_8_5 = callPackage ../applications/science/logic/coq {
     version = "8.5pl3";
-    inherit (ocamlPackages) ocaml findlib lablgtk;
```

Comment:
I see, so indeed this makes the `all-packages.nix`. If this can be extended to other packages as well?